Makefile

```diff
@@ -196,7 +199,7 @@ prepare-release: bump-release generate fmt vet manifests bundle
 	$(MAKE) bundle CHANNELS=alpha,stable DEFAULT_CHANNEL=alpha
 
 bump-release:
-	sed -i 's/version string = "v\(.*\)"/version string = "v$(VERSION)"/g' pkg/version/version.go
+	sed -i '' 's/version string = "v\(.*\)"/version string = "v$(VERSION)"/g' pkg/version/version.go
```

Comment:
This doesn't work for linux

```bash
▶ make bump-release VERSION=dep
# sed -i 's/version string = "v\(.*\)"/version string = "vdep"/g' pkg/version/version.go
sed -i '' 's/version string = "v\(.*\)"/version string = "vdep"/g' pkg/version/version.go
sed: can't read s/version string = "v\(.*\)"/version string = "vdep"/g: No such file or directory
make: *** [Makefile:200: bump-release] Error 2
```

Reply:
Change dropped in https://github.com/3scale-ops/saas-operator/pull/105#event-4805762798.

(I ended installing gnu-sed in my mbp, was cleaner solution than adding logic to the makefile)